components/raftstore/src/store/peer.rs

```diff
@@ -1279,6 +1285,7 @@ where
                 if progress.matched >= truncated_idx {
                     let (_, pending_after) = self.peers_start_pending_time.swap_remove(i);
                     let elapsed = duration_to_sec(pending_after.elapsed());
+                    RAFT_PEER_PENDING_DURATION.observe(elapsed);
```

Comment:
I don't think it's necessary. Histogram will reset automatically after fetching.

Reply:
Why? This is used to record the finish time. Without this line, We can't know the max pending duration accurately.